posthog/temporal/workflows/logger.py

```diff
@@ -230,7 +231,7 @@ def attempt_to_fetch_workflow_info() -> Info | None:
     """
     try:
         workflow_info = temporalio.workflow.info()
-    except RuntimeError:
+    except (RuntimeError, _NotInWorkflowEventLoopError):
```

Comment:
I'm moving all logging to activities in #18581, so I'm getting rid of this function. I'll merge that and rebase this.